wzdx/standard_to_enhanced/__init__.py

```diff
@@ -0,0 +1,3 @@
+from wzdx.standard_to_enhanced.cotrip_translator import wzdx_creator
```

Comment:
Does this get confused all three having the same name for import? From this standpoint how do we know we're using cotrip/icon/navjoy version of the wzdx_creator?

Reply:
the __init__ (as I understand it) essentually makes it easier to import the packages. So instead of having to explicitely import the whole chain, you can just import wzdx and them call wzdx.standard_to_enhanced.cotrip-translator.wzdx_creator("message")

Reply:
Ah so with this init, it's still requiring the full name to call the wzdx_creator method and you wouldn't call
`import wzdx`
and then 
`wzdx.wzdx_creator("message")`

If that's the case, then this makes sense. Does that mean that only the `wzdx_creator` methods are being made available then? For instance you couldn't call `wzdx.standard_to_enhanced.cotrip-translator.validate_alert(alert)`?

Reply:
i believe it just means if you wanted to call validate_alert, you would have to do 
```
from wzdx.standard_to_enhanced import cotrip_translator
cotrip_translator.validate_alert()
```
but you could not do 
```
import wzdx
wzdx.standard_to_enhanced.cotrip_translator.validate_alert()
```